src/reducer/wishList-reducer.js

```diff
@@ -0,0 +1,28 @@
+const wishListReducer = (state, action) => {
+
+  switch (action.type) {
+    case "LOAD_DATA":
+      return {
+        ...state,
+        wishListItem: [...action.payload],
+      };
+
+    case "ADD_PRODUCT_TO_WISHLIST":
+      return {
+        ...state,
+        wishListItem: [...action.payload],
+        wishListCount: state.wishListCount + 1,
+      };
+    case "DELETE_PRODUCT_FROM_WISHLIST":
+      return {
+        ...state,
+        wishListItem: [...action.payload],
+        wishListCount: state.wishListCount - 1,
```

Comment:
IMO, you `wishlistCount` can be easily calculated from `wishListItem.length` method, there's no need for having a separate property for count in the state. I did it the same way at first but later i realised, that in some cases, it creates more bugs.